ReSharper.FSharp/test/data/features/generate/overrides/Repr - Empty - Class 03 - Comment.fs.gold

```diff
@@ -8,4 +8,5 @@
 // ${SELECT0:ToString():System.String}
 
 type T() = // comment
+
```

Comment:
This is definitely not expected 🙂

Reply:
Should be better now. Some other leading empty lines are gone as well.